[2024-05-18T19:17:59.073+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:17:59.090+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-04-19T01:00:00+00:00 [queued]>
[2024-05-18T19:17:59.097+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-04-19T01:00:00+00:00 [queued]>
[2024-05-18T19:17:59.098+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:17:59.109+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2023-04-19 01:00:00+00:00
[2024-05-18T19:17:59.115+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1181) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:17:59.117+0000] {standard_task_runner.py:63} INFO - Started process 1183 to run task
[2024-05-18T19:17:59.117+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2023-04-19T01:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmpqb0o5rth']
[2024-05-18T19:17:59.119+0000] {standard_task_runner.py:91} INFO - Job 267: Subtask hello_task
[2024-05-18T19:17:59.162+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2023-04-19T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:17:59.238+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2023-04-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-19T01:00:00+00:00'
[2024-05-18T19:17:59.240+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:17:59.241+0000] {logging_mixin.py:188} INFO - 2023-04-19/
[2024-05-18T19:17:59.242+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:17:59.243+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:17:59.251+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2023-04-19T01:00:00+00:00, execution_date=20230419T010000, start_date=20240518T191759, end_date=20240518T191759
[2024-05-18T19:17:59.291+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:17:59.306+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:17:59.308+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
